kythe/go/platform/tools/viewindex/viewindex.go

```diff
@@ -70,6 +72,17 @@ func main() {
 		if err := json.NewEncoder(out).Encode(idx); err != nil {
 			log.Fatalf("Error encoding JSON: %v", err)
 		}
+	} else if *printFile != "" {
+		for _, f := range idx.Files {
+			if f.Info.GetDigest() == *printFile {
+				if _, err := os.Stdout.Write(f.Content); err != nil {
+					log.Fatal(err)
+				}
+				return
+			}
+		}
+		fmt.Fprintf(os.Stderr, "File digest %q not found\n", *printFile)
```

Comment:
The rest of the code uses log.Fatal for this (which, modulo timestamps, is basically equivalent).

Reply:
Fix'd